webapps-schema/src/main/resources/schema/elasticsearch/create/index/camunda-tenant.json

```diff
@@ -14,6 +14,9 @@
       "name": {
         "type": "keyword"
       },
+      "description": {
+        "type": "text"
```

Comment:
💭 Just thinking out loud here, as this is a text field it means the field is effectively parsed/tokenised and stored in the index in a way that allows partial matching, I think thats fine here as it would match the future usage of the field of finding all the tenants where a description contains `something`. The alternative is to use `keyword` which doesn't perform the parsing so I'm not sure of the implications there.

Reply:
Right but `keyword` has problems too, for example failing on very large values. Not sure which is better here tbh.

Reply:
Yea, like I say I was just thinking out loud, I think for our purposes `text` is fine (happy for you to seek other opinions though!)